web/src/ui/pages/home/Home.tsx

```diff
@@ -43,7 +44,25 @@ export default function Home(props: Props) {
     const statsCases = config.home.statistics.catgegories as Array<availableStat>;
 
     type availableUseCase = "declareReferent" | "editSoftware" | "addSoftwareOrService";
-    const useCases = config.home.usecases.catgegories as Array<availableUseCase>;
+    type UseCaseConfig = {
+        enabled: boolean;
+        labelLinks: any;
+        buttonEnabled: boolean;
+        buttonLink: string;
+    };
+    type UsesCaseConfig = Record<availableUseCase, UseCaseConfig>;
+
+    const configUseCases: UsesCaseConfig = config.home.usecases;
+    const keys: Array<availableUseCase> = Object.keys(
+        configUseCases
+    ) as Array<availableUseCase>;
+
+    const useCases: Array<availableUseCase> = keys.reduce(
+        (accumulator: Array<availableUseCase>, key: availableUseCase) => {
+            return configUseCases[key].enabled ? accumulator.concat([key]) : accumulator;
+        },
+        []
+    );
```

Comment:
En fait c'est juste un filtre ça :

```suggestion
    const useCases = keys.filter(useCaseName => configUseCases[useCaseName].enabled)
```